### DISCUSSION TITLE
USR-W630 integration

### DISCUSSION THREAD

**Discussion by Andromeda1966** [kellerza/sunsynk#321] [General]
Is there a guide on how to configure USR-W630 with Deye/Sunsynk integration ?
Thanks in advance

**Comment by hamid-elaosta** [comment]
Did you ever figure this out? I got mine working and I wonder if it's worth adding some details here or to the documentation for kellerza's page since it's a little lacking in details?

I'm happy to write up the details somewhere, it was fairly straightforward in the end, but I absolutely needed a 120Ohm resistor across the A and B terminals for it to work reliably.
